### PR TITLE
Add NN Loss Functions to Paddle Frontend #15053

### LINKED ISSUE #15053: Add `paddle.nn.functional.loss` NN Loss Functions to PaddlePaddle Frontend
Add [NN Loss Functions](https://www.paddlepaddle.org.cn/documentation/docs/en/api/index_en.html) to PaddlePaddle Frontend

> Please keep in mind that the proper way to link an issue to this list is to comment "- [ ] #issue_number" while the issue's title only includes the name of the function you've chosen.

- [ ] binary_cross_entropy
- [ ] binary_cross_entropy_with_logits
- [ ] cross_entropy
- [ ] dice_loss
- [ ] hsigmoid_loss
- [ ] kl_div
- [ ] l1_loss
- [ ] log_loss
- [ ] margin_ranking_loss
- [ ] mse_loss
- [ ] nll_loss
- [ ] npair_loss
- [ ] sigmoid_focal_loss
- [ ] smooth_l1_loss
- [ ] softmax_with_cross_entropy
- [ ] margin_cross_entropy
- [ ] square_error_cost
- [ ] ctc_loss
- [ ] hinge_embedding_loss
- [ ] cosine_embedding_loss
- [ ] multi_label_so
- [ ] triplet_margin_with
- [ ] tripl
- [ ] so

### PR DESCRIPTION
**Pull request by Aryan8912**
# PR Description

## Related Issue

Close #15053 

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/conv.py [hunk 1]
```diff
@@ -185,3 +185,66 @@ def conv3d_transpose(
...
+                # Weighted mean for soft labels
+                pass
+            else:
+                # Weighted mean for hard labels
+                pass
+
+
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def dice_loss(input, label, epsilon=0.00001, name=None):
```
Comment (juliagsy) [inline ivy/functional/frontends/paddle/nn/functional/conv.py:250]: is this added by mistake? it seems to me it's missing a definition, please either add it back if it was there before or remove it as it is not part of your function, thanks!

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Thanks for the great work so far, please also add its corresponding test in the corresponding folder in `ivy_tests` and fix all test failures, thanks!

**Comment by juliagsy** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
